[SELS-PREP] Setup TailwindCss

balagtasangelo opened the pull request:
[Commands]
            None
 [Pre-conditions]
           None
[Expected Output]
             None
[Notes]
                This is only a setup for the css using Tailwind.

balagtasangelo committed:
[SELS-PREP] Setup TailwindCss

jeremiahC commented on resources/css/app.css:
```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```
add new line and just leave a space

jeremiahC commented on public/mix-manifest.json:
```diff
@@ -0,0 +1,4 @@
+{
+    "/js/app.js": "/js/app.js",
+    "/css/app.css": "/css/app.css"
+}
```
two space indentation for json

```suggestion
{
  "/js/app.js": "/js/app.js",
  "/css/app.css": "/css/app.css"
}
```

balagtasangelo committed:
Update public/mix-manifest.json

Commands]
None
[Pre-conditions]
None
[Expected Output]
None
[Notes]
This is only a setup for the css using Tailwind.

Co-authored-by: jeremiahC <jeremiah.caballero.jc@gmail.com>

balagtasangelo committed:
[SELS-PREP] Added new line for app.css file

balagtasangelo committed:
Update app.css
